gravitee-am-test/api/management/models/PreviewResponse.ts

```diff
@@ -0,0 +1,82 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Gravitee.io - Access Management API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
```

Comment:
`mapValues` is not used. Noticed this unused import on other interfaces as well. Let's remove them if possible just to keep the code base cleaner.